unstoppable-ios-app/domains-manager-ios/Modules/WalletDetails/WalletDetailsViewController.swift

```diff
@@ -290,3 +290,20 @@ extension WalletDetailsViewController {
     }
     
 }
+
+@available(iOS 17.0, *)
```

Comment:
Would be nice to keep this mock data for preview outside of `WalletDetailsViewController` file